clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchResponse.java

```diff
@@ -49,9 +49,9 @@
      *
      *  UNKNOWN_TOPIC_OR_PARTITION (3)  <- only for request v0
      *  OFFSET_LOAD_IN_PROGRESS (14)
-     *  NOT_COORDINATOR_FOR_CONSUMER (16)
```

Comment:
It looks like the offset fetch is the only request where we return NOT_COORDINATOR_FOR_GROUP for that case. It's a little unclear which error code should be preferred, but I would tend to think that NOT_COORDINATOR_FOR_GROUP would be the right one since it would force the client to rediscover the coordinator, which is probably what we want when we're shutting down. On the other hand, when we're starting up, we'd want to use GROUP_COORDINATOR_NOT_AVAILABLE. What do you think?